python/oneflow/nn/graph/graph.py

```diff
@@ -1296,7 +1293,9 @@ def finish_compile_and_init_runtime(self):
                 compiled_job_str = self._c_nn_graph.get_current_job_str()
                 self._compiled_job_proto = job_pb.Job()
                 self._compiled_job_proto.ParseFromString(compiled_job_str)
-
+                self.__print(
+                    0, 2, lambda: f"{self.name} with operators:\n" + self.__repr__()
```

Comment:
这里的 2 改成 1 吧，然后试下 debug(1)

Reply:
以上次的 nansum 为例的 debug(1) 输出：
```
(GRAPH:Graph_0:Graph) start building graph.
(GRAPH:Graph_0:Graph) start building graph builders of parameters and buffers.
(GRAPH:Graph_0:Graph) end building graph builders of parameters and buffers.
(GRAPH:Graph_0:Graph) start building graph inputs.
(INPUT:_Graph_0_input.0.0_2:tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
       sbp=(oneflow.sbp.split(dim=0),), size=(8, 8, 0, 16, 24),
       dtype=oneflow.float32))
(GRAPH:Graph_0:Graph) end building graph inputs.
(GRAPH:Graph_0:Graph) start building graph modules.
(MODULE:model:Module())
(INPUT:_model_input.0.0_2:tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
       sbp=(oneflow.sbp.split(dim=0),), is_lazy='True', size=(8, 8, 0, 16, 24),
       dtype=oneflow.float32))
tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
       sbp=(oneflow.sbp.split(dim=0),), is_lazy='True', size=(8, 8, 0, 16, 24),
       dtype=oneflow.float32)
(OUTPUT:_model_output.0.0_2:tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
       sbp=(oneflow.sbp.split(dim=0),), is_lazy='True', size=(8, 0, 16, 24),
       dtype=oneflow.float32))
(GRAPH:Graph_0:Graph) end building graph modules.
(GRAPH:Graph_0:Graph) start building graph outputs.
(OUTPUT:_Graph_0_output.0.0_2:tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
       sbp=(oneflow.sbp.split(dim=0),), is_lazy='True', size=(8, 0, 16, 24),
       dtype=oneflow.float32))
(GRAPH:Graph_0:Graph) end building graph outputs.
tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
       sbp=(oneflow.sbp.split(dim=0),), is_lazy='True', size=(8, 8, 0, 16, 24),
       dtype=oneflow.float32)(GRAPH:Graph_0:Graph) start building graph with compile passes.

                                                                         
[1/60][GraphCompile]Graph_0 InsertPinnedIdentityOpPass
                                                                         
[2/60][GraphCompile]Graph_0 EliminateDeadNodesPass
                                                                         
[3/60][GraphCompile]Graph_0 NormalizationExponentialAverageAutoTic
                                                                         
[4/60][GraphCompile]Graph_0 AutoMixedPrecision
                                                                         
[5/60][GraphCompile]Graph_0 PruneDependOpPass
                                                                         
[6/60][GraphCompile]Graph_0 PruneAmpWhiteIdentityOpPass
                                                                         
[7/60][GraphCompile]Graph_0 OptimizerPlacementOptimizationPass
                                                                         
[8/60][GraphCompile]Graph_0 FuseAddToOutputPass
                                                                         
[9/60][GraphCompile]Graph_0 DynamicLossScaleSchedulePass
                                                                         
[10/60][GraphCompile]Graph_0 AutoTrainStep
                                                                         
[11/60][GraphCompile]Graph_0 AutoLearningRate
                                                                         
[12/60][GraphCompile]Graph_0 QuantAwareTraining
                                                                         
[13/60][GraphCompile]Graph_0 GenerateOptimizerOpConfs
                                                                         
[14/60][GraphCompile]Graph_0 PrunePinnedIdentityOpPass
                                                                         
[15/60][GraphCompile]Graph_0 ReplaceEmbeddingOps
                                                                         
[16/60][GraphCompile]Graph_0 SequentialOneEmbeddingOpsPass
                                                                         
[17/60][GraphCompile]Graph_0 FuseEmbeddingShuffleInteractionPass
                                                                         
[18/60][GraphCompile]Graph_0 FuseBCEReduceMeanFwBwPass
                                                                         
[19/60][GraphCompile]Graph_0 AddSspVariableProxy
                                                                         
[20/60][GraphCompile]Graph_0 CheckpointingPass
                                                                         
[21/60][GraphCompile]Graph_0 CudnnFusedNormalizationAddReluPass
                                                                         
[22/60][GraphCompile]Graph_0 PruneCastToStaticShapeOpsPass
                                                                         
[23/60][GraphCompile]Graph_0 FuseAddToOutputPass
                                                                         
[24/60][GraphCompile]Graph_0 FuseConsecutiveAddPass
                                                                         
[25/60][GraphCompile]Graph_0 IndexedSlicesOptimizerRewritePass
                                                                         
[26/60][GraphCompile]Graph_0 SplitSparseSoftmaxCrossEntropyOpPass
                                                                         
[27/60][GraphCompile]Graph_0 DoParallelCastBeforeWideningTypeCast
                                                                         
[28/60][GraphCompile]Graph_0 FuseCastScalePass
                                                                         
[29/60][GraphCompile]Graph_0 PruneParallelCastOpsPass
                                                                         
[30/60][GraphCompile]Graph_0 FuseUpdateOpsPass
                                                                         
[31/60][GraphCompile]Graph_0 FuseModelUpdateCastOpsPass
                                                                         
[32/60][GraphCompile]Graph_0 MultiTensorModelUpdatePass
                                                                         
[33/60][GraphCompile]Graph_0 FixPipelineStageIdPass
                                                                         
[34/60][GraphCompile]Graph_0 PipelineBufferPass
                                                                         
[35/60][GraphCompile]Graph_0 AutoParallelPass
                                                                         
[36/60][GraphCompile]Graph_0 DelayVariableOpExecutionPass
                                                                         
[37/60][GraphCompile]Graph_0 CutlassConvTuningWarmupPass
                                                                         
[38/60][GraphCompile]Graph_0 DumpVariableInfoPass
                                                                         
[39/60][GraphCompile]Graph_0 DumpBlobParallelConfPass
(GRAPH:Graph_0:Graph) end building graph with compile passes.
(GRAPH:Graph_0:Graph) start re-building graph outputs for optimizatioin.
(GRAPH:Graph_0:Graph) end re-building graph outputs for optimizatioin.
(GRAPH:Graph_0:Graph) building graph Done! Cost time: 0.6s.

(GRAPH:Graph_0:Graph) start building plan.
                                                                         
[40/60][GraphCompile]Graph_0 GroupBoxingByDstParallel
                                                                         
[41/60][GraphCompile]Graph_0 BoxingWithMiddleNodes
                                                                         
[42/60][GraphCompile]Graph_0 SetCtrl
                                                                         
[43/60][GraphCompile]Graph_0 AutoPrependTick
                                                                         
[44/60][GraphCompile]Graph_0 AddTickForTimeShape
                                                                         
[45/60][GraphCompile]Graph_0 AutoSourceAndSinkTick
                                                                         
[46/60][GraphCompile]Graph_0 CriticalSectionTick
                                                                         
[47/60][GraphCompile]Graph_0 SystemOpFillJobNamePass
                                                                         
[48/60][GraphCompile]Graph_0 DumpBlobParallelConfPass
                                                                         
[49/60][GraphCompile]Graph_0 CheckAndLogOpGraph
Graph_0 with operators:
(GRAPH:Graph_0:Graph): (
  (CONFIG:config:GraphConfig(training=False, ))
  (INPUT:_Graph_0_input.0.0_2:tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
         sbp=(oneflow.sbp.split(dim=0),), size=(8, 8, 0, 16, 24),
         dtype=oneflow.float32))
  (MODULE:model:Module()): (
    (INPUT:_model_input.0.0_2:tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
           sbp=(oneflow.sbp.split(dim=0),), is_lazy='True', size=(8, 8, 0, 16, 24),
           dtype=oneflow.float32))
    (OUTPUT:_model_output.0.0_2:tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
           sbp=(oneflow.sbp.split(dim=0),), is_lazy='True', size=(8, 0, 16, 24),
           dtype=oneflow.float32))
    (GraphModule:model()): (
      (OPERATOR: model-constant-0() -> (model-constant-0/out_0:(sbp=(S(0)), size=(8, 8, 0, 16, 24), dtype=(oneflow.float32))), placement=(oneflow.placement(type="cuda", ranks=[0, 1])))
      (OPERATOR: model-cast-1(model-constant-0/out_0:(sbp=(S(0)), size=(8, 8, 0, 16, 24), dtype=(oneflow.float32))) -> (model-cast-1/out_0:(sbp=(S(0)), size=(8, 8, 0, 16, 24), dtype=(oneflow.bool))), placement=(oneflow.placement(type="cuda", ranks=[0, 1])))
      (OPERATOR: model-masked_fill-2(_Graph_0_input.0.0_2/out:(sbp=(S(0)), size=(8, 8, 0, 16, 24), dtype=(oneflow.float32)), model-cast-1/out_0:(sbp=(S(0)), size=(8, 8, 0, 16, 24), dtype=(oneflow.bool))) -> (model-masked_fill-2/out_0:(sbp=(S(0)), size=(8, 8, 0, 16, 24), dtype=(oneflow.float32))), placement=(oneflow.placement(type="cuda", ranks=[0, 1])))
      (OPERATOR: model-reduce_nansum-3(model-masked_fill-2/out_0:(sbp=(S(1)), size=(8, 8, 0, 16, 24), dtype=(oneflow.float32))) -> (model-reduce_nansum-3/output_tensor_0:(sbp=(S(0)), size=(8, 0, 16, 24), dtype=(oneflow.float32))), placement=(oneflow.placement(type="cuda", ranks=[0, 1])))
    )
  )
  (OPERATOR: _Graph_0_input.0.0_2() -> (_Graph_0_input.0.0_2/out:sbp=(S(0)), size=(8, 8, 0, 16, 24), dtype=(oneflow.float32)), placement=(oneflow.placement(type="cuda", ranks=[0, 1])))
  (OPERATOR: _Graph_0_output.0.0_2(model-reduce_nansum-3/output_tensor_0:sbp=(S(0)), size=(8, 0, 16, 24), dtype=(oneflow.float32)) -> (_Graph_0_output.0.0_2/out:sbp=(S(0)), size=(8, 0, 16, 24), dtype=(oneflow.float32)), placement=(oneflow.placement(type="cuda", ranks=[0, 1])))
  (OPERATOR: empty-4() -> (empty-4/out_0:(sbp=(B), size=(4, 0, 16, 24), dtype=(oneflow.float32))), placement=(oneflow.placement(type="cuda", ranks=[0])))
  (OUTPUT:_Graph_0_output.0.0_2:tensor([], placement=oneflow.placement(type="cuda", ranks=[0, 1]),
         sbp=(oneflow.sbp.split(dim=0),), is_lazy='True', size=(8, 0, 16, 24),
         dtype=oneflow.float32))
)
*****************************************
Setting OMP_NUM_THREADS environment variable for each process to be 1 in default, to avoid your system being overloaded, please further tune the variable for optimal performance in your application as needed. 
*****************************************
Killing subprocess 3354320
Killing subprocess 3354321
```